packages/core/core/src/Parcel.js

```diff
@@ -437,6 +450,78 @@ export default class Parcel {
     logger.info({origin: '@parcel/core', message: 'Taking heap snapshot...'});
     return this.#farm.takeHeapSnapshot();
   }
+
+  async transform(options: ParcelTransformOptions): Promise<Array<Asset>> {
+    if (!this.#initialized) {
+      await this._init();
+    }
+
+    let projectRoot = nullthrows(this.#resolvedOptions).projectRoot;
+    let request = createAssetRequest({
+      ...options,
+      filePath: toProjectPath(projectRoot, options.filePath),
+      optionsRef: this.#optionsRef,
+      env: createEnvironment({
+        ...options.env,
+        loc:
+          options.env?.loc != null
+            ? {
+                ...options.env.loc,
+                filePath: toProjectPath(projectRoot, options.env.loc.filePath),
+              }
+            : undefined,
+      }),
+    });
+
+    let res = await this.#requestTracker.runRequest(request, {force: true});
```

Comment:
Should we use `force` here? My understanding is that `force` means the cache is ignored.

Reply:
I can update that, but it was originally copied from the intiial branch, I will leave this Q to @devongovett on if this is required?

Reply:
Just changed it to a configurable value. Follows the config of `shouldCacheDisabled` from the main config.
I guess this could be helpful to debug transform problems.